src/test/java/org/jabref/logic/bibtex/FileFieldWriterTest.java

```diff
@@ -47,8 +50,16 @@ public void parseCorrectOnlineInput() throws MalformedURLException {
     public void parseFaultyOnlineInput() {
         String input = ":htt\\://arxiv.org/pdf/2010.08497v1:PDF";
         String inputURL = "htt://arxiv.org/pdf/2010.08497v1";
-        List<LinkedFile> expected = Collections.singletonList(new LinkedFile("", Path.of(inputURL), "PDF"));
 
+        if (OS.WINDOWS) {
+            // on Windows, a ":" in a path cannot be used
+            // java.nio.file.InvalidPathException: Illegal char <:> at index 3: htt://arxiv.org/pdf/2010.08497v1
+            // Thus, we assert that statement and return
+            assertThrows(InvalidPathException.class, () -> Path.of(inputURL));
+            return;
+        }
+
+        List<LinkedFile> expected = Collections.singletonList(new LinkedFile("", Path.of(inputURL), "PDF"));
```

Comment:
What is `FileFieldParser.parse(input)` returning on Windows?

Reply:
It also returns an exception. Changed the `assertThrows` accordingly.

The issue is at `field = new LinkedFile(entry.get(0), Path.of(entry.get(1)), entry.get(2));`, where `Path.of(...)` is called. Future work? 😇

Reply:
We have something like LinkedFile.isOnlineLink as check in some places

Reply:
I think you found a bug then. The string should always be parsed to a LinkedFile, even if it's not a valid path or url. The fix should be simple as well: just introduce a constructor accepting the path also a string, and change the filefieldparser to catch the invalidpathexception as well.